spec/requests/api/groups_spec.rb

```diff
@@ -3,8 +3,9 @@
 describe API::API, api: true  do
   include ApiHelpers
 
-  let(:user1) { create(:user) }
+  let(:user1) { create(:user, can_create_group: false) }
   let(:user2) { create(:user) }
+  let(:user3) { create(:user) }
```

Comment:
Well, I see you use it. But in the diff I don;t see a reason you can't use user2. can you please explain in a little bit more wording why it is needed? I have about 100+ PR's that I need to review. I can't go and source dive all the time. Thats the reason I ask those questions. 
Its not my intention to insult you or anything like that. If you feel that way, I'm sorry.

I need to be sure that a PR is good before we can merge it. If I have the feeling it is not good I'm forced to close it without merging. That wouldn't help you, nor the GitLab project.